charts/fluent-operator/values.yaml

```diff
@@ -22,9 +24,10 @@ operator:
       requests:
         cpu: 50m
         memory: 64Mi
-  container:
-    repository: "kubesphere/fluent-operator"
-    tag: "latest"
+  image:
```

Comment:
aligned naming convention on how charts define the image values. also see the update in the template itself.